### PR TITLE
updated customer summary

### PR DESCRIPTION
**Pull request by Asadupramod012**
updated customer summary.

### CODE REVIEW

File: apps/customer/serializers.py [hunk 1]
```diff
@@ -97,7 +97,49 @@ def get_shipping_address(self, obj):
...
+#         # Prepare the addresses in the desired format
+#         customer_addresses = []
+#         if billing_address and shipping_address:
+#             customer_addresses.append({
+#                 "billing_address": f"{billing_address.address}, {billing_address.city_id.city_name}, {billing_address.state_id.state_name}, {billing_address.country_id.country_name}, {billing_address.pin_code}, Phone: {billing_address.phone}",
+#                 "shipping_address": f"{shipping_address.address}, {shipping_address.city_id.city_name}, {shipping_address.state_id.state_name}, {shipping_address.country_id.country_name}, {shipping_address.pin_code}, Phone: {shipping_address.phone}"
+#             })
+        
+#         return customer_addresses
+
```
Comment (QodeFarm) [inline apps/customer/serializers.py:141]: Remove the above commented code.

File: apps/customer/serializers.py [hunk 2]
```diff
@@ -106,35 +148,49 @@ class Meta:
...
+    def get_customer_details(self, obj):
         addresses = CustomerAddresses.objects.filter(customer_id=obj.customer_id)
+        
+        email = None
+        phone = None
         billing_address = None
         shipping_address = None
         
         for address in addresses:
+            if email is None:
```
Comment (QodeFarm) [inline apps/customer/serializers.py:160]: email is already None on 154, so does this condition is needed?

File: apps/customer/serializers.py [hunk 3]
```diff
@@ -106,35 +148,49 @@ class Meta:
...
+                email = address.email
+            if phone is None:
```
Comment (QodeFarm) [inline apps/customer/serializers.py:162]: phone is already None on line 155, so does this condition is needed?

File: apps/customer/serializers.py (hunk 1 above)
Comment (Asadupramod012) [reply apps/customer/serializers.py:141]: Done

**Pull request merged**